test/benchmarks/crypto_callbacks.mjs

```diff
@@ -0,0 +1,87 @@
+import crypto from 'node:crypto';
```

Comment:
Instead of duplicating these here, can we add the driver as a dev dependency and use the hooks defined in the driver?

Reply:
I appreciate the desire to de-dupe but I think that's overkill. I imagine we'll delete the ones in the driver after switching to native crypto, which would make this script no longer work with a `^` caret dependency on the driver. Additionally, those callbacks aren't public driver API so even if they don't get removed an innocent refactor could break an import here.

Reply:
Oh but this looks like a perfect opportunity to move these backs into the bindings source code (i.e. next to index.ts)? I still believe that moving these into the driver was a mistake, if we need them here in the tests that's just yet another reason to move them back

Reply:
> I imagine we'll delete the ones in the driver after switching to native crypto

There's probably going to be a need for JS crypto callbacks until something like the alternative approach mentioned in the ticket here is implemented – We can't use libmongocrypt's native crypto integration on macOS and Windows in mongosh because we need to be able to toggle the FIPS mode switch for those.

Reply:
> move these backs into the bindings source code (i.e. next to index.ts)?

Not opposed, we still need to leave the callbacks in the driver so bindings versions below the next release continue working. 

> We can't use libmongocrypt's native crypto integration on macOS and Windows

I think reintroducing them to the src will fit better in https://github.com/mongodb-js/mongodb-client-encryption/pull/18 since currently I've essentially prevented callbacks from working unless you build from libmongocrypt src using nocrypto. 

I will ping you on that PR when it is ready. 

> something like the alternative approach mentioned in the ticket here is implemented

What's this referencing? Are you referring to what https://github.com/mongodb-js/mongodb-client-encryption/pull/18 is attempting, using builtin OpenSSL?

Reply:
The idea was to use the opensll that's bundled and exposed with Nodejs on all platforms for the crypto hooks (i.e., c++ crypto callbacks).  That would take the place of https://github.com/mongodb-js/mongodb-client-encryption/pull/18.

Reply:
See https://jira.mongodb.org/browse/NODE-5455?focusedCommentId=5590599&focusedId=5590599&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-5590599

Reply:
After this morning's discussion I think we have what we need here for this PR. In the following PR where we implement C++ callbacks we can also bring in the JS callbacks if we desire. Is that accurate?

Reply:
We can't bring the JS callbacks over, that's breaking.  But other than that, yes.

Reply:
> We can't bring the JS callbacks over, that's breaking

Why is it breaking (for mongodb-client-encryption)?